backend/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+db:
+  image: postgres:9.5
+web:
+  build: .
+  command: python manage.py runserver 0.0.0.0:8000
+  volumes:
+    - .:/opt/akvo-dash
```

Comment:
My thinking was that we need to have access to the Python project code while it's being developed. Would we not need to build a new container on each code change otherwise?